po/ar.po

```diff
@@ -0,0 +1,347 @@
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:59
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:64
+msgid "Translation updates"
+msgstr "تحديثات الترجمة"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:53
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:84
+msgid "Performance and stability"
+msgstr "الأداء والاستقرار"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:65
+msgid "Fix special characters in search results"
+msgstr "إصلاح الأحرف الخاصة في نتائج البحث"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:70
+msgid "Fixes a crash that sometimes occurs while uninstalling an extension"
+msgstr "يعمل على إصلاح العطل الذي يحدث أحيانًا أثناء إلغاء تثبيت أحد الامتدادات"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:75
+msgid "The first feature update to extension-manager. Highlights include:"
+msgstr "أول تحديث للميزة لمدير الامتداد. ومن أبرز ما جاء فيها:"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:77
+msgid "Dark theme and support for overriding the system colour scheme"
+msgstr "مظهر داكن ودعم لتجاوز نظام ألوان النظام"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:78
+msgid "Shows screenshots in extension listings"
+msgstr "يظهر لقطات في قوائم الامتداد"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:79
+msgid "Shell version compatibility check"
+msgstr "التحقق من توافق إصدار شل"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:80
+msgid "Localised into seven languages"
+msgstr "مترجمة إلى سبع لغات"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:81
+msgid "Sort search results"
+msgstr "فرز نتائج البحث"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:82
+msgid "Global extension on/off toggle"
+msgstr  "تبديل تشغيل / إيقاف الامتداد العام"
+
+#: data/com.mattjakeman.ExtensionManager.appdata.xml.in.in:83
+msgid "Displays user and system extensions separately"
+msgstr "يعرض امتدادات المستخدم والنظام بشكل منفصل"
+
+#: src/exm-application.c:106
+msgid "Extension Manager (Development)"
+msgstr "مدير الامتداد (تطوير)"
+
+#: src/exm-application.c:107
+msgid "Extension Manager"
+msgstr "مدير الامتداد"
+
+#. TRANSLATORS: 'Name <email@domain.com>' or 'Name https://website.example'
+#: src/exm-application.c:117
+msgid "translator-credits"
+msgstr "Omar TS ots@duck.com"
+
+#: src/exm-application.c:118
+msgid ""
+"A very simple tool for browsing, downloading, and managing GNOME shell "
+"extensions."
+msgstr "أداة بسيطة للغاية لتصفح وتنزيل وإدارة امتدادات جنوم شل "
+"."
+
+#: src/exm-application.c:124
+msgid "Project Homepage"
+msgstr "صفحة المشروع الرئيسية"
+
+#: src/exm-browse-page.blp:7
+msgid "Search for extensions"
+msgstr "بحث عن الامتدادات"
+
+#: src/exm-browse-page.blp:8
+msgid ""
+"Enter a keyword to search 'extensions.gnome.org' for GNOME Shell Extensions."
+msgstr "أدخل كلمة أساسية للبحث في "extension.gnome.org" عن امتدادات جنوم شل."
```

Comment:
The correct URL is 'extensions' and not 'extension'. Also I'm not 100% sure, but the quotation marks here should probably be escaped with a backslash:

```suggestion
msgstr "أدخل كلمة أساسية للبحث في \"extensions.gnome.org\" عن امتدادات جنوم شل."
```